Languages/Python/nice_implementations/tests/trie/test_trie.py

```diff
@@ -1,4 +1,4 @@
-from src.trie.trie import Trie
+from ...src.trie.trie import Trie
```

Comment:
MAYBE: Could you leave it as it was before - the tests are supposed to be run from the root of the project